packages/typespec-ts/src/utils/modelUtils.ts

```diff
@@ -1867,3 +1902,31 @@ export function isBodyRequired(parameter: HttpOperationParameters) {
     ? true
     : false;
 }
+
+export function getCollectionFormat(
+  context: SdkContext,
+  param:
+    | HttpOperationPathParameter
+    | HttpOperationQueryParameter
+    | HttpOperationHeaderParameter
+): string | undefined {
+  const type = param.param;
+  const encode = getEncode(context.program, param.param);
+  if (
+    type.type.kind === "Model" &&
+    isArrayModelType(context.program, type.type)
+  ) {
+    if (param.explode) {
+      return "multi";
+    }
+    switch (encode?.encoding) {
+      case "ArrayEncoding.pipeDelimited":
```

Comment:
do we still have pipes ssv csv?

Reply:
we do, please check this out https://github.com/microsoft/typespec/blob/main/packages/http-specs/specs/parameters/collection-format/main.tsp